src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-weight: 300px;
+
+}
+
+.search_bar {
+  font-family: "Avenir-Heavy", sans-serif;
+  height: 70px;
+  width: 100%;
+  border-radius: 4px;
+  margin-bottom: 20px;
+  display: block;
+  text-indent: 10%;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  font-size: 16px;
+  font-weight: bold;
+  outline: none;
+}
+
+.search_bar_small {
+  font-family: "Avenir-Heavy", sans-serif;
+  height: 42px;
+  width: 100%;
+  border-radius: 4px;
+  margin-bottom: 20px;
+  display: block;
+  text-indent: 10%;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  font-size: 14px;
+  font-weight: bold;
+  outline: none;
```

Comment:
put here only styles that differ from .input (checklist point 2)
the same on other similar cases